seedvault/src/main/java/com/solanamobile/seedvault/SeedVault.java

```diff
@@ -49,7 +51,7 @@ public static boolean isAvailable(@NonNull Context context, boolean allowSimulat
 
         for (PermissionInfo permission : pi.permissions) {
             if (WalletContractV1.PERMISSION_SEED_VAULT_IMPL.equals(permission.name) &&
-                    ((permission.flags & PermissionInfo.PROTECTION_FLAG_PRIVILEGED) != 0)) {
+                    ((permission.getProtectionFlags() & PROTECTION_FLAG_PRIVILEGED) != 0)) {
```

Comment:
@J909 https://github.com/solana-mobile/seed-vault-sdk/actions/runs/4171225225/jobs/7231733394#step:5:589

Reply:
```
/home/runner/work/seed-vault-sdk/seed-vault-sdk/seedvault/src/main/java/com/solanamobile/seedvault/SeedVault.java:54: Error: Call requires API level 28 (current min is 23): android.content.pm.PermissionInfo#getProtectionFlags [NewApi]
                    ((permission.getProtectionFlags() & PROTECTION_FLAG_PRIVILEGED) != 0)) {
                                 ~~~~~~~~~~~~~~~~~~
```

Reply:
Can probably just use the `protectionLevel` field directly: https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/content/pm/PermissionInfo.java;l=361